PKNetworkActivityController.podspec

```diff
@@ -1,14 +1,14 @@
 Pod::Spec.new do |s|
   s.name                      = 'PKNetworkActivityController'
   s.module_name               = 'PKNetworkActivityController'
-  s.version                   = '0.0.1'
+  s.version                   = '0.0.2'
 s.summary                     = 'A simple and lightweight network activity indicator controller written in Swift, exclusively for iOS 8 and above.'
-  s.homepage                  = 'https://github.com/pkluz/PKNetworkActivityController'
+  s.homepage                  = 'https://github.com/dimohamdy/PKNetworkActivityController'
```

Comment:
Please change this in the fork you wish to submit the PR for, otherwise I cannot accept the merge as the entry would be incorrect for this framework.